### DISCUSSION TITLE
many itmes only display metric units of measure, need automatic conversion.

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#20] [General]
Many items don't display alternate units of measure for weight. I am fairly certain that in fitbit dashboard automatically brought up alternate units of measures (or if not brought up it would allow you to enter it) if at least one was entered. If there were grams it would be offered in oz's and lbs or if it was lb's it would be grams or kgs. Can such functionality be implemented?

![image](https://github.com/user-attachments/assets/a87965a9-3ccf-49cd-b64d-198ccefbdebb)

![image](https://github.com/user-attachments/assets/cced0ae2-bfb1-40f2-b609-37a9890eebc4)

**Comment by jlai** [comment]
I've updated it so it always adds common units to the serving size picker: oz, grams, kg, lbs
